examples/simulation-pytorch/utils.py

```diff
@@ -58,14 +57,10 @@ def test(net, testloader, device: str):
     return loss, accuracy
 
 
-def get_mnist(data_path: str = "./data"):
-    """Download MNIST and apply transform."""
+def get_mnist_transforms():
+    """Get transformation for MNIST dataset."""
```

Comment:
I'm not the biggest fan of recreating that composite transform all the time. I think we can also just define as access. But it might be more of a design choice.

Reply:
I did it as we discussed 👍